tests/functional/test_states_data/provider_and_host/non_multi_success/config.yaml

```diff
@@ -5,26 +5,25 @@
   version: 1
 
   actions:
-    multi:
+    state_changing_action:
```

Comment:
As I recall all non-empty files should have a license comment at the top of the file?

Reply:
Yep. But i think we will fix it in a separate PR